src/parameters/Parameters.jl

```diff
@@ -51,6 +75,15 @@ end
 
 Base.eltype(::ClimaAtmosParameters{FT}) where {FT} = FT
 
+thermodynamics_params(ps::ATCP) = ps.thermodynamics_params
+surface_fluxes_params(ps::ATCP) = ps.surface_fluxes_params
+
+# Forward TurbulenceConvection parameters
+const TCPS = TurbulenceConvectionParameters
+for fn in fieldnames(TCPS)
+    @eval $(fn)(ps::ATCP) = ps.$(fn)
+end
```

Comment:
Could we add parameter forwarding for `turbconv_params` like we do for the thermo params?
Then we can do
`detr_coeff = CAP.detr_coeff(params)` instead of 
```
turbconv_params = CAP.turbconv_params(params)
`detr_coeff = CAP.detr_coeff(turbconv_params)`
```

Reply:
```
for var in fieldnames(TCPS)
    @eval $var(ps::ACAP) = $var(turbconv_params(ps))
 ```